react/components/ExtraData.tsx

```diff
@@ -98,6 +152,13 @@ const ExtraData: React.VFC<Props> = ({
       return
```

Comment:
Look at the line 151 (above this one, but I can't select it :v)

Could you change the 'new' magical string to NEW_ADDRESS_VALUE as well?

Reply:
this isn't the same as `NEW_ADDRESS_VALUE`, this is referring whether or not it is a new card or a saved card

Reply:
it is actually an enum, so it is being type-checked (a typo would've been caught by typescript)

Reply:
oh ok ok cool